templates/external_auth_api.conf.erb

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
I'll note that we'd like to move to [EPP](https://www.puppet.com/docs/puppet/7/lang_template_epp.html) instead of ERB. I won't block on it, but I like that you can pass an explicit context. From the example:
```puppet
epp('example/example.epp', { 'logfile' => "/var/log/ntp.log" })
```

Then you can simply use `$logfile` in the template, rather than using `scope.lookupvar()`. In my experience it gives more reliability that templates actually render correctly, especially if you add in data types at the top.

Reply:
I'd leave the current approach. I guess transition to EPP will be in batch anyway for all current files at some point...